src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue9006.cs

```diff
@@ -18,14 +27,15 @@ public Issue9006(TestDevice testDevice) : base(testDevice)
 		[Category(UITestCategories.Compatibility)]
 		public void ClickingOnTabToPopToRootDoesntBreakNavigation()
```

Comment:
This test is failing on iOS:
<img width="932" alt="image" src="https://github.com/user-attachments/assets/df2955a6-cfcb-41f3-b7bd-c2a013e7517e">

Reply:
Resolved the CI failure, also ensured and commited the code changes. 
![image](https://github.com/user-attachments/assets/4d1a0dfc-792d-4ff0-8cc2-2903a25cd6e1)
![image](https://github.com/user-attachments/assets/b9064b2c-3bf5-4b7f-ba92-d414bdaf7ad3)